src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="../src/images/imac.jpeg"
+        alt="apple"
+      />
+      <div class="card__product-description">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+      <div class="card__product-code"><p>Product code: 195434</p></div>
```

Comment:
The same as above, this wrapper is redundant, fix it everywhere
```suggestion
      <p class="card__product-code">Product code: 195434</p>
```